.github/workflows/prbot.yml

```diff
@@ -1,18 +1,157 @@
 name: PR Robot
-on:
-- pull_request
-
+on: [pull_request]
 jobs:
-  pr-robot:
+  build:
+    name: "Build"
     runs-on: ubuntu-latest
     env:
       GITHUB_CREDENTIALS: ${{ secrets.GITHUB_TOKEN }}
     steps:
-    - uses: actions/checkout@v2
-      with:
+      - uses: actions/checkout@v2
+        with:
           fetch-depth: 0
-    - uses: actions/setup-node@v1
-      with:
-          node-version: '14'
-    - run: npm i
-    - run: npm run pr:report
+      - uses: actions/setup-node@v2
+        with:
+          node-version-file: '.nvmrc'
+      - uses: actions/cache@v2
+        with:
+          path: |
+            node_modules
+            */*/node_modules
+          key: ${{ runner.os }}-${{ hashFiles('**/package-lock.json') }}
+      - run: |
+          npm ci
+          npx lerna bootstrap
+      - run: npm run build
```

Comment:
I tried to make the build run in parallel.
* I tried adding the `--stream` flag, the build took about the same amount of time if not longer to build. I guess that makes sense since each builds slow down the other builds that are running in parallel.
* I tried manually specifying which builds to run first with `--scope` and using `--parallel`, which shaved off about 20 seconds compared to my control test, which I believe is negligible enough on a ~6 mins process. I would rather keep the command simple/automatic.

Reply:
If it's only 20s, I agree with keeping it simple. Still, we will bring that build time down in other ways :)